### PR TITLE
Updated gitignore

### PR DESCRIPTION
**Pull request by MatteoWohlrapp**
Updated gitignore to include files generated on the cluster

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,13 @@
 .ropeproject
-./zdpath
-./zoutput
+/zdpath
```
Comment (smilesun) [inline .gitignore:2]: /zdpath is absolution path?

Comment (MatteoWohlrapp) [reply .gitignore:2]: ./zdpath does not work for me on the cluster. It does not ignore the directory then

Comment (smilesun) [reply .gitignore:2]: ok. I do not understand why the CI passed. I thought / means the root directory of a computer?

Comment (MatteoWohlrapp) [reply .gitignore:2]: But I think in the case of gitignore it means the root directory of the git project

**Pull request merged**